tests/ui/abi/debug.rs

```diff
@@ -7,8 +7,14 @@
 // This pattern is prepared for when we account for alignment in the niche.
 //@ normalize-stderr: "(valid_range): [1-9]\.\.=(429496729[0-9]|1844674407370955161[0-9])" -> "$1: $$NON_NULL"
 // Some attributes are only computed for release builds:
+//@ add-core-stubs
 //@ compile-flags: -O
-#![feature(rustc_attrs)]
+//@ revisions: generic riscv64
+//@ [riscv64] only-riscv64
+//@ [generic] ignore-riscv64
+
+#![feature(rustc_attrs,no_core)]
+#![no_core]
```

Comment:
You do not need two instances of `#![no_core]` and `#![feature(no_core)]`
```suggestion
#![feature(rustc_attrs)]
```